lib/ansible/module_utils/basic.py

```diff
@@ -2087,13 +2098,19 @@ def run_command(self, args, check_rc=False, close_fds=True, executable=None, dat
             # the communication logic here is essentially taken from that
             # of the _communicate() function in ssh.py
 
-            stdout = ''
-            stderr = ''
+            stdout = b('')
+            stderr = b('')
```

Comment:
[comment on code that was pre-existing to this pr and doesn't need to be revised now]

My preference would be to rename these to something like stdout_data/stderr_data. Having vars named 'stdout' and
'stderr' floating around code dealing with subprocess/popen/select seem error prone, especially since the 'stdout' var here is not a pipe or file object.